packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -98,6 +98,17 @@ export const schema = {
     }),
     viewableBy: ['guests'],
   },
+  createdBy: {
```

Comment:
most instances of foreignKeyField I see are for fields with an "Id" at the end of their name, implying a convention. Doublechecking that this is should be "createdBy" and not either "userId" or something like "createdByUserId"?

Reply:
This should've had the key and `idFieldName` matching, which was actually bad. Made it consistent (both `userId`).